go/libraries/doltcore/table/editor/table_edit_session.go

```diff
@@ -47,14 +47,14 @@ func CreateTableEditSession(root *doltdb.RootValue, props TableEditSessionProps)
 	return &TableEditSession{
 		Props:      props,
 		root:       root,
-		tables:     make(map[string]*SessionedTableEditor),
+		tables:     make(map[string]*sessionedTableEditor),
 		writeMutex: &sync.RWMutex{},
 	}
 }
 
-// GetTableEditor returns a SessionedTableEditor for the given table. If a schema is provided and it does not match the one
+// GetTableEditor returns a sessionedTableEditor for the given table. If a schema is provided and it does not match the one
```

Comment:
Nit: for the comment, I wouldn't reference the implementation type by name since we return the `TableEditor` interface.